pkgs/misc/cups/drivers/cups-pdf-to-pdf/default.nix

```
@@ -0,0 +1,64 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cups
+, makeBinaryWrapper
+, coreutils
+, nixosTests
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cups-pdf-to-pdf";
+  version = "unstable-2021-12-22";
+  src = fetchFromGitHub {
+    owner = "alexivkin";
+    repo = "CUPS-PDF-to-PDF";
+    rev = "c14428c2ca8e95371daad7db6d11c84046b1a2d4";
+    hash = "sha512-FP+GGq3hVAXrLzidAsp3Q1WNzZxrIDaQHN36+RTu1W09FlOfZnIVdw/uJHBMWSm03HIBPSwQmpmpW8KqRv6izQ==";
+  };
+  buildInputs = [ cups ];
```

```suggestion
  version = "unstable-2021-12-22";

  src = fetchFromGitHub {
    owner = "alexivkin";
    repo = "CUPS-PDF-to-PDF";
    rev = "c14428c2ca8e95371daad7db6d11c84046b1a2d4";
    hash = "sha512-FP+GGq3hVAXrLzidAsp3Q1WNzZxrIDaQHN36+RTu1W09FlOfZnIVdw/uJHBMWSm03HIBPSwQmpmpW8KqRv6izQ==";
  };

  buildInputs = [ cups ];
```

Please insert some new lines between options in the package.